### PR TITLE
Add edit resourceset endpoint

### PR DESCRIPTION
**Pull request by abhishekdharmapurikar**
Adding endpoints to edit protectionset

### CODE REVIEW

File: pkg/polaris/graphql/infinity-k8s/infinity_k8s.go [hunk 1]
```diff
@@ -66,6 +66,13 @@ type AddK8sProtectionSetConfig struct {
 	RSType              string   `json:"rsType"`
 }
 
+// UpdateK8sProtectionSetConfig defines the input parameters required to update
+// a ProtectionSet.
+type UpdateK8sProtectionSetConfig struct {
```
Comment (gqi128) [inline pkg/polaris/graphql/infinity-k8s/infinity_k8s.go:71]: name can't be updated?

**Review by gqi128 — APPROVED** [review]

File: pkg/polaris/graphql/infinity-k8s/infinity_k8s_test.go [hunk 2]
```diff
@@ -128,6 +128,23 @@ func TestIntegration(t *testing.T) {
...
+		infinityk8s.UpdateK8sProtectionSetConfig{
+			Definition:  "{\"includes\": [{\"resource\": \"Deployment\"}]}",
+		},
+	)
+	if err != nil {
+		t.Error(err)
+		return
+	}
+	if updateResp != true {
+		t.Errorf("update failed, %v", updateResp)
```
Comment (rajath26) [inline pkg/polaris/graphql/infinity-k8s/infinity_k8s_test.go:144]: %v -> %+v

Comment (rajath26) [reply pkg/polaris/graphql/infinity-k8s/infinity_k8s_test.go:144]: No need to return here?

File: pkg/polaris/graphql/infinity-k8s/infinity_k8s.go (hunk 1 above)
Comment (abhishekdharmapurikar) [reply pkg/polaris/graphql/infinity-k8s/infinity_k8s.go:71]: no, since its also a crd and crd can't change names.

**Pull request merged**